fs/vfs/fs_rename.c

```diff
@@ -90,7 +90,7 @@ static int pseudorename(FAR const char *oldpath, FAR struct inode *oldinode,
         {
           inode_release(newinode);
           ret = OK;
-          goto errout; /* Bad naming, this is not an error case. */
+          goto errout; /* Same name, this is not an error case. */
```

Comment:
Should it be considered an error? At least on Linux when trying to move a directory to itself we see an error reported:
```
$ mv a a
mv: cannot move 'a' to a subdirectory of itself, 'a/a'
```

Reply:
@acassis the message is reported by mv command, not function Here is my test on WSL2:
```
$ cat test.c
#include <stdio.h>

int main(int argc, char *argv[])
{
    return rename(argv[1], argv[2]);
}
$ gcc test.c 
$ ./a.out a.out a.out 
$ echo $?
0
```

Reply:
Yeah. Probably will need to update `cmd_mv` in NSH

Reply:
Could be